### PR TITLE
Fashion Files - New hair, screens, markings, and more

### PR DESCRIPTION
**Pull request by Sinfulbehaviors**
Sometimes, personally, I feel almost let down by the lack of certain hairstyles, the appearances of certain styles, and just an overall want for more customization. So, I've spent the last month and a half coming up with, spriting, and coding in new cosmetic items, starting with human and IPC cosmetics. 

Forum post [here.](https://forums.aurorastation.org/topic/17269-fashion-files-a-whole-lotta-new-cosmetics/)

### CODE REVIEW

**Comment by Sinfulbehaviors** [comment]
!wip

File: code/modules/mob/abstract/new_player/sprite_accessories.dm [hunk 1]
```diff
@@ -4181,6 +4305,12 @@ Follow by example and make good judgement based on length which list to include
 			icon_state = "hand_panel_l"
 			body_parts = list(BP_L_HAND)
 
+	autopsy
```
Comment (KingOfThePing) [inline code/modules/mob/abstract/new_player/sprite_accessories.dm:4308]: This is more of a personal opinion but I don't see this being logical to take ever. MAYBE think about leaving this one out.

File: code/modules/mob/abstract/new_player/sprite_accessories.dm [hunk 2]
```diff
@@ -4181,6 +4305,12 @@ Follow by example and make good judgement based on length which list to include
...
+		name = "Autopsy Scar"
+		icon_state = "autopsyscar"
+		body_parts = list(BP_CHEST)
+		species_allowed = list(/datum/species/human, /datum/species/human/offworlder, /datum/species/machine/shell, /datum/species/diona, /datum/species/diona/coeu, /datum/species/tajaran, /datum/species/tajaran/zhan_khazan, /datum/species/tajaran/m_sai, /datum/species/unathi)
```
Comment (KingOfThePing) [inline code/modules/mob/abstract/new_player/sprite_accessories.dm:4312]:
```suggestion
		species_allowed = list(/datum/species/human, /datum/species/human/offworlder, /datum/species/diona, /datum/species/diona/coeu, /datum/species/tajaran, /datum/species/tajaran/zhan_khazan, /datum/species/tajaran/m_sai, /datum/species/unathi)
```

File: .gitignore [hunk 3]
```diff
@@ -17,4 +17,9 @@ __pycache__/
 *$py.class
 /.atom-build.json
 .vscode/
-.vs/
\ No newline at end of file
+.vs/
+config/example/admins.txt
```
Comment (KingOfThePing) [inline .gitignore:21]: You should not edit this file. Revert this back to original.

**Review by KingOfThePing — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 4]
```diff
@@ -17,4 +17,9 @@ __pycache__/
...
+config/example/admins.txt
```
Comment (KingOfThePing) [inline .gitignore:22]: ditto

File: .gitignore [hunk 5]
```diff
@@ -17,4 +17,9 @@ __pycache__/
...
+config/example/alienwhitelist.txt
```
Comment (KingOfThePing) [inline .gitignore:23]: ditto

File: .gitignore [hunk 6]
```diff
@@ -17,4 +17,9 @@ __pycache__/
...
+config/example/admins.txt
```
Comment (KingOfThePing) [inline .gitignore:24]: ditto

File: .gitignore [hunk 7]
```diff
@@ -17,4 +17,9 @@ __pycache__/
...
+config/example/config.txt
```
Comment (KingOfThePing) [inline .gitignore:25]: ditto

**Review by KingOfThePing — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 7 above)
Comment (Sinfulbehaviors) [reply .gitignore:25]: Everything in gitignore are for my test server and the test server alone, I'll remove them in the next push.

File: code/modules/mob/abstract/new_player/sprite_accessories.dm (hunk 1 above)
Comment (Sinfulbehaviors) [reply code/modules/mob/abstract/new_player/sprite_accessories.dm:4308]: If other devs agree, I'll remove it.

File: code/modules/mob/abstract/new_player/sprite_accessories.dm (hunk 2 above)
Comment (Sinfulbehaviors) [reply code/modules/mob/abstract/new_player/sprite_accessories.dm:4312]: Yeah, I forgot to remove them from the copy paste. Will commit.

**Review by KingOfThePing — APPROVED** [review]

**Comment by listerla** [comment]
You seem to have offset a number of preview icons; offworlder_preview.dmi and human_preview.dmi namely. 

IPC_breathe has a stray white pixel for several frames in the forward-facing icon.

You left a template each in facial_hair.dmi and markings.dmi, and two in hair.dmi. 

You've removed hair_punk_halfshaved_alt and hair_punk_halfshaved_bun_alt.

**Comment by NonQueueingMatt** [comment]
Inactive.

**Pull request closed without merging**